com.unity.render-pipelines.universal/ShaderLibrary/SSAO.hlsl

```diff
@@ -249,62 +305,62 @@ void SampleDepthNormalView(float2 uv, out float depth, out float3 normal, out fl
 // Distance-based AO estimator based on Morgan 2011
 // "Alchemy screen-space ambient obscurance algorithm"
 // http://graphics.cs.williams.edu/papers/AlchemyHPG11/
-float4 SSAO(Varyings input) : SV_Target
+half4 SSAO(Varyings input) : SV_Target
 {
     UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
     float2 uv = input.uv;
 
     // Parameters used in coordinate conversion
-    float2 p11_22, p13_31;
-    float3x3 camTransform = (float3x3)_CameraViewProjections[unity_eyeIndex]; // camera viewProjection matrix
+    half3x3 camTransform = (half3x3)_CameraViewProjections[unity_eyeIndex]; // camera viewProjection matrix
 
     // Get the depth, normal and view position for this fragment
     float depth_o;
-    float3 norm_o;
-    float3 vpos_o;
+    half3 norm_o;
+    half3 vpos_o;
     SampleDepthNormalView(uv, depth_o, norm_o, vpos_o);
 
     // This was added to avoid a NVIDIA driver issue.
-    float randAddon = uv.x * 1e-10;
-
-    float rcpSampleCount = rcp(SAMPLE_COUNT);
-    float ao = 0.0;
-    for (int s = 0; s < int(SAMPLE_COUNT); s++)
+    const half rcpSampleCount = half(rcp(SAMPLE_COUNT));
+    half ao = 0.0;
+    for (int s = 0; s < SAMPLE_COUNT; s++)
     {
-        #if defined(SHADER_API_D3D11)
-            // This 'floor(1.0001 * s)' operation is needed to avoid a DX11 NVidia shader issue.
-            s = floor(1.0001 * s);
-        #endif
-
         // Sample point
-        float3 v_s1 = PickSamplePoint(uv, randAddon, s); // (kchang) should we rotate this "random" vector to world space?
+        half3 v_s1 = PickSamplePoint(uv, s); // (kchang) should we rotate this "random" vector to world space?
```

Comment:
Is there an answer to this question?
Since there are multiple spaces, we should document the spaces for position variables. Either use our space 
postfix convention like WS, CS etc. or just add comments.

Reply:
Agree! 
I've removed the comment as we shouldn't have discussions in the codebase and noted it down for the future work.
Allow me to tweak the variable naming with proper postfixes in the followup PR.